modules/local/propr/propd/main.nf

```diff
@@ -0,0 +1,33 @@
+process PROPR_PROPD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/bioconductor-limma_r-ggplot2_r-propr:209490acb0e524e3' :
+        'community.wave.seqera.io/library/bioconductor-limma_r-ggplot2_r-propr:17abd3f137436739' }"
+
+    input:
+    tuple val(meta), path(count), path(samplesheet), val(contrast_variable), val(reference), val(target)
```

Comment:
This is very un-nf-core, and doesn't match the analagous differential modules. I think this will need to be split for nf-core

Reply:
even though why this is not recommended by nf-core?

Reply:
Ahh- has there been a change? Could you point at that guideline?

Reply:
not sure if there a change/guideline for that...that is why i asked because i was not sure why you say this is un-nf-core.